test/Npgsql.Specification.Tests/Npgsql.Specification.Tests.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
```

Comment:
Should we jump this to `netcoreapp3.0` to align with #2396?

Reply:
Makes sense! Removed the setting altogether to simply have it picked up from Directory.Build.props, like the main test project.